graviti/schema/builtin.py

```diff
@@ -38,6 +71,11 @@ def __init__(self, **kwargs: Any) -> None:
 
         super().__init__(**kwargs)
 
+    def _to_pyarrow(self) -> "paDataType":
+        return BuiltinExtension(
+            self.__class__.__name__, self._pyarrow_type_factory(), **self.params.to_pyobj()
```

Comment:
`to_pyobj`? normal to_pyobj will contain "type" field?
the to_pyobj for complex type may contain `imports` field?
are you sure it is a proper method here?